courses/free-courses-bn.md

```diff
@@ -289,7 +289,7 @@
 
 #### Laravel
 
-* [Restapi Bangla tutorial \| Laravel & Lumen Api development](https://www.youtube.com/playlist?list=PLbC4KRSNcMnoQONzuNtFlhEzegTYadoBY) - Learn Hunter
+* [laravel 10 bangla tutorial latest version | laravel bangla tutorial | laravel basic to advanced from laravel official documentation](https://www.youtube.com/playlist?list=PLbC4KRSNcMnrY78JyoI8c0pk-reuSw8ff) - Learn Hunter
```

Comment:
```suggestion
* [laravel 10 bangla tutorial latest version](https://www.youtube.com/playlist?list=PLbC4KRSNcMnrY78JyoI8c0pk-reuSw8ff) - Learn Hunter
```
Although our policy is to take the title from the resource, occasionally truncation of a very long title is an improvement.